cristin-biobank/build.gradle

```diff
@@ -8,6 +8,7 @@ dependencies {
     implementation libs.nva.core
     implementation libs.nva.language
     implementation libs.aws.lambda.java.core
+    implementation project(path: ':cristin-project')
```

Comment:
you should not depend on another module other than cristin-commons. Is there a reason module cristin-project is required?

Reply:
Reuse of classes... DateInfo etc.
Move to common ?

Reply:
i think that it best to avoid circular dependencies and not having to build other modules when changing one